src/backend/PortabilityService.AnalysisEngine/Controllers/AnalyzeController.cs

```diff
@@ -89,25 +83,11 @@ private Task<AnalyzeResult> AnalyzeRequestAsync(AnalyzeRequest analyzeRequest, s
                 //TODO: invoke the real analysis engine to do the work
                 //return _requestAnalyzer.AnalyzeRequest(analyzeRequest, submissionId);
```

Comment:
Probably would be better to encapsulate this in a dummy _requestAnalyzer and register that so this code won't have to change later. Include a comment that the embedded resource itself should be removed if the code is removed